### DISCUSSION TITLE
Is Vercel is capable of handling of complex UI Web Apps?

### DISCUSSION THREAD

**Discussion by PrimeQA-Dev** [vercel/community#3964] [Feature Question]
### Question

I wanted to know that is Vercel is capable of handling complex UI webapps, as it is serverless. It shouldnot cause any issue on UI sluggishness.

**Comment by amyegan** [comment]
Hey @PrimeQA-Dev. Yes, I have seen many complex sites on Vercel. 

Check out the templates page for some examples: https://vercel.com/templates

**Comment by PrimeQA-Dev** [comment]
Thanks,
Is there anything differently we need to handle our UI specific to vercel?
Or as usual we can start developing based on our setup and requirements.

Thanks,
Sachin Patel
